anu_lms.module

```diff
@@ -178,3 +180,31 @@ function anu_lms_anu_lms_sw_scripts_alter(&$data) {
     watchdog_exception('anu_lms', $exception);
   }
 }
+
+/**
+ * Implements hook_entity_insert().
+ *
+ * Adds an empty page to a lesson if there are no pages.
+ */
+function anu_lms_entity_insert(EntityInterface $entity) {
+  if ($entity->bundle() !== 'module_lesson') {
+    return;
+  }
+
+  $lesson_content = $entity->get('field_module_lesson_content');
+
+  if (!empty($lesson_content->getValue())) {
+    return;
+  }
+
+  $paragraph = Paragraph::create([
+    'type' => 'lesson_section',
+  ]);
+  $paragraph->save();
+
+  $lesson_content->appendItem([
```

Comment:
I think this code works only because the objects are passed by reference. Although it's not mandatory, but would be better to just set entity's field value directly.